src/Shyim/PluginInstaller.php

```diff
@@ -133,7 +140,11 @@ private static function downloadPlugin($name, $version)
         });
 
         if (empty($plugin)) {
-            throw new \RuntimeException(sprintf('Plugin with name "%s" is not available in your Account. Please buy the plugin first', $name));
+            if (self::$silentFail) {
```

Comment:
Can you create a new method something like throwException and there then the if case.
And very important you have forgot the **return** to stop the next execution